go.mod

```diff
@@ -76,16 +75,15 @@ require (
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.7.0
 	github.com/opencontainers/go-digest v1.0.0-rc1
-	github.com/opencontainers/runc v1.0.0-rc10
-	github.com/opencontainers/selinux v1.3.3
+	github.com/opencontainers/runc v1.0.0-rc9.0.20200519182809-b207d578ec2d
```

Comment:
is this switching to an untagged release of runc?

Reply:
@liggitt yep, we talked about it here - https://github.com/google/cadvisor/pull/2469#issuecomment-631435135 unfortunately runc rc11 is a while away

Reply:
it looks like this picks up some changes (https://github.com/opencontainers/runc/commit/4b4bc995ad4ee4679ffbe38f1672be01c24256fc) that were later reverted (https://github.com/opencontainers/runc/commit/a67dab0ac2dd3829858e35ba643613a1e358ba87)... it's unclear to me that this mid-tag version is suitable for us to pick up

Reply:
I've updated to a newer commit.  Fixes for cgroup v2 are still going to happen in libcontainer, it will likely take some more iterations to get it right.

Reply:
> it will likely take some more iterations to get it right

Can you expand on this comment?

Is Kubernetes using cgroups v2 now? If so, it sounds like we're picking up changes we know are still in-progress, which doesn't seem desireable. If not, is it important for us to get this update now?

Reply:
sure, there is still a lot of work going on in the runc repository for supporting cgroup v2, the refactor affects also cgroup v1.  I've opened this PR as a checkpoint to avoid having to introduce more changes later when a release is finally cut (hopefully in time for 1.19) and have some time to use it

Reply:
We have a chicken and egg situation here. We want to validate the work done in runc for cgroups v2 so far with k8s, so we can go back and address any gaps or issues.

Reply:
> the refactor affects also cgroup v1

That is my main concern. It's unclear to me what state this PR leaves the kubernetes master branch in, since the v2 changes we want to validate are entangled with changes to v1 code we currently depend on. I'm not in favor of experimental bumps of master to unstable versions of dependencies.

Reply:
this is not really  a chicken and egg, if the work is experimental it can be tested with a patchset that is not in master.